recipes/miniscript/all/conanfile.py

```diff
@@ -0,0 +1,60 @@
+from conan import ConanFile
+from conan.tools.files import get, copy
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+
+import os
+
+required_conan_version = ">=1.53.0"
+
+class MiniscriptConan(ConanFile):
+    name = "miniscript"
+    description = "modern, elegant, easy to learn, and easy to embed in your own C# or C++ projects."
+    license = "LGPL-3.0"
```

Comment:
```suggestion
    license = "MIT"
```
https://github.com/JoeStrout/miniscript/blob/172bea8e762c96b1b2a5ea743228ff6a58702cc3/LICENSE